### ISSUE TITLE
Bug belvédère

### ISSUE THREAD

**Issue by YoyoKeeper16** [JulienNigon/SpellcasterUniversity-Bug-Report#793]
J'ai eu plusieurs fois la carte Belvédère, mais je n'arrive pas à la placer, bien que j'essaie tout au long de la partie à de nombreux endroits.

**Comment by CherryWingNLR** [comment]
C'est pas un bug. Tu dois t'assurer d'avoir 2 case de haut, et surtout, une entrée depuis l'étage supérieur. Par exemple, tu ne peux pas le placer à côté de l'étage de la Bibliothèque ou de la salle temporelle car il n'y a pas d'ouverture. Un indice : Regarde s'il y a des balcons

**Comment by kaori01** [comment]
C'est ça :)
Une photo si ça peut aider... en regardant bien tu vois que le passage se fait par le second étage...

![image](https://user-images.githubusercontent.com/60629516/74233148-73e95e00-4cca-11ea-926b-3c4914df359f.png)

**Comment by Neelan22** [comment]
Oui, il faut une case libre au dessus et au dessous de la passerelle. On va clarifier ça.
